docs/api/QUIC_CREDENTIAL_CONFIG.md

```diff
@@ -15,50 +15,162 @@ typedef struct QUIC_CREDENTIAL_CONFIG {
         QUIC_CERTIFICATE* CertificateContext;
         QUIC_CERTIFICATE_FILE* CertificateFile;
         QUIC_CERTIFICATE_FILE_PROTECTED* CertificateFileProtected;
+        QUIC_CERTIFICATE_PKCS12* CertificatePkcs12;
     };
     const char* Principal;
     void* Reserved; // Currently unused
     QUIC_CREDENTIAL_LOAD_COMPLETE_HANDLER AsyncHandler; // Optional
+    QUIC_ALLOWED_CIPHER_SUITE_FLAGS AllowedCipherSuites;// Optional
 } QUIC_CREDENTIAL_CONFIG;
 ```
 
 # Members
 
-`Type`
+#### `Type`
```

Comment:
We usually follow a slightly different format for flags (see https://github.com/microsoft/msquic/blob/main/docs/api/StreamSend.md). But this can be updated in a future PR.